### PR TITLE
feat: dark mode update #2

### PR DESCRIPTION
**Pull request by corquaid**
Description
---
Continuing Dark Mode implementation

- [x] BaseNode main views
- [x] Wallet main views

Not included:
- linked modal views, e.g. connect to aurora
- wallet transactions
  
Motivation and Context
---
#36 partial

How Has This Been Tested?
---

**Screenshots**

_BaseNode setup/paused_
Screenshot 2022-07-08 at 20 59 30
Screenshot 2022-07-08 at 20 59 48

_BaseNode running_
Screenshot 2022-07-08 at 21 00 21
Screenshot 2022-07-08 at 21 00 33

_Wallet setup_
Screenshot 2022-07-08 at 21 01 23
Screenshot 2022-07-08 at 21 01 33

_Wallet running_
Screenshot 2022-07-08 at 21 02 53
Screenshot 2022-07-08 at 21 04 23

### CODE REVIEW

**Review by tomaszantas — DISMISSED** [review]
Looks good!

File: applications/launchpad/gui-react/src/components/Select/types.ts [hunk 1]
```diff
@@ -28,6 +28,8 @@ export type Option = {
  *    @property {string} [color] - text color of the value
  *    @property {string} [backgroundColor] - background color
  *    @property {(open?: boolean) => string} [borderColor] - allows to set different border color for opened and closed state
+ *  @property {Object} [label] - label styles override
```
Comment (tarnas14) [inline applications/launchpad/gui-react/src/components/Select/types.ts]:
```suggestion
 *    @property {Object} [label] - label styles override
```

File: applications/launchpad/gui-react/src/components/TBot/HelpComponents/Wallet/index.tsx [hunk 2]
```diff
@@ -31,3 +31,12 @@ export const NoteAboutVerificationPeriod = () => {
     </>
   )
 }
+
+export const TariWalletIdHelp = (
+  <>
```
Comment (tarnas14) [inline applications/launchpad/gui-react/src/components/TBot/HelpComponents/Wallet/index.tsx]: looks like an unnecessary fragment

**Review by tarnas14 — DISMISSED** [review]

File: applications/launchpad/gui-react/src/components/Select/types.ts (hunk 1 above)
Comment (corquaid) [reply applications/launchpad/gui-react/src/components/Select/types.ts]: Fixed.

File: applications/launchpad/gui-react/src/components/TBot/HelpComponents/Wallet/index.tsx (hunk 2 above)
Comment (corquaid) [reply applications/launchpad/gui-react/src/components/TBot/HelpComponents/Wallet/index.tsx]: Removed.

**Review by tomaszantas — APPROVED** [review]

**Pull request merged**